### PR TITLE
Create the footer using tailwind css

### PR DESCRIPTION
**Pull request by vishwaikon**
…in external CSS stylesheet were replaced with Tailwind CSS classes. The code to import the external stylesheet was commented to prevent style conflicts.

## Purpose

The purpose of this PR is to develop the footer of the website and replace the Antd styles with Tailwind CSS which is a part of fixing issue #50. 

## Goals
Tailwind classes were used to enhance efficiency and readability and the structure.

## Approach
The CSS flexbox was used and Tailwind CSS classes were used to achieve the Antd styles. The code to import the external stylesheet was commented to prevent conflicts in styles and to be used if required.

## Screenshots
image

##  Checklist
- [X ] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [X ] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [X ] My code follows the style guidelines of this project
- [X ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/components/Layout/Footer/Footer.tsx [hunk 1]
```diff
@@ -1,167 +1,314 @@
...
-  FacebookFilled,
-  GithubOutlined,
-  InstagramFilled,
-  LinkedinFilled,
-  SlackOutlined,
-  TwitterOutlined,
-  YoutubeFilled,
-} from '@ant-design/icons';
-import { Button, Col, Row, Space, Typography } from 'antd';
+// import styles from './Footer.module.css';
```
Comment (anjula-sack) [inline src/components/Layout/Footer/Footer.tsx]:
```suggestion
```

File: src/components/Layout/Footer/Footer.tsx [hunk 2]
```diff
@@ -1,167 +1,314 @@
...
-        <Text className={styles.footerMainTitle}>Let`s Rethink Education!</Text>
-        <Text className={styles.footerSecondaryTitle}>
+  <>
+    <div className="wrapper flex justify-around flex-col sm:flex-row ">
+      <div className="col ml-8">
+        <div className="footerMainTitle font-sans text-4xl text-darkmod-blue font-bold mb-12">
+          Let`s Rethink Education!
+        </div>
+        <div className="footerSecondaryTitle font-sans text-base text-darkmod-blue font-normal">
+          {' '}
```
Comment (anjula-sack) [inline src/components/Layout/Footer/Footer.tsx]:
```suggestion
```

File: src/components/Layout/Footer/Footer.tsx [hunk 3]
```diff
@@ -1,167 +1,314 @@
...
+                className="w-7"
+                viewBox="0 -6 512 512"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <path fill="#475a96" d="M0 0h512v500H0z" />
+                <path
+                  d="M375.717 112.553H138.283c-8.137 0-14.73 6.594-14.73 14.73v237.434c0 8.135 6.594 14.73 14.73 14.73h127.826V276.092h-34.781v-40.28h34.781v-29.705c0-34.473 21.055-53.244 51.807-53.244 14.73 0 27.391 1.097 31.08 1.587v36.026l-21.328.01c-16.725 0-19.963 7.947-19.963 19.609v25.717h39.887l-5.193 40.28h-34.693v103.355h68.012c8.135 0 14.73-6.596 14.73-14.73V127.283c-.001-8.137-6.596-14.73-14.731-14.73z"
+                  fill="#ffffff"
+                />
+              </svg>
```
Comment (anjula-sack) [inline src/components/Layout/Footer/Footer.tsx]: Let's create separate components for svgs so we can manage them easily. create a folder called assets->svgs and keep the svgs there.

File: src/components/Layout/Footer/Footer.tsx [hunk 4]
```diff
@@ -1,167 +1,314 @@
...
+                  <title>youtube [#ffffff]</title>{' '}
+                  <desc>Created with Sketch.</desc> <defs> </defs>{' '}
+                  <g
+                    id="Page-1"
+                    stroke="none"
+                    stroke-width="1"
+                    fill="none"
+                    fill-rule="evenodd"
+                  >
+                    {' '}
```
Comment (anjula-sack) [inline src/components/Layout/Footer/Footer.tsx]: Remove this unwanted spaces
```suggestion
```

File: src/components/Layout/Footer/Footer.tsx [hunk 5]
```diff
@@ -1,167 +1,314 @@
...
+          href="https://sefglobal.org/"
+          className=" sefLink active:text-darkmod-blue font-sans font-normal"
+        >
           {' '}
           Sustainable Education Foundation - SEF.
-        </a>
-      </Text>
-    </Col>
-  </Row>
+        </a>{' '}
```
Comment (anjula-sack) [inline src/components/Layout/Footer/Footer.tsx]:
```suggestion
        </a>
```

**Review by anjula-sack — COMMENTED** [review]
Good job! @vishwaikon I added some comments. Also there are lint fails. Run npm run lint to check them. Also keep the PR titles short.

File: src/components/Layout/Footer/Footer.tsx (hunk 3 above)
Comment (vishwaikon) [reply src/components/Layout/Footer/Footer.tsx]: Sure, I will create

File: src/components/Layout/Footer/Footer.tsx [hunk 6]
```diff
@@ -1,167 +1,313 @@
...
+      </div>
+    </div>
+  </>
 );
 export default FooterComponent;
```
Comment (mayura-andrew) [inline src/components/Layout/Footer/Footer.tsx:144]: add EOF here after line number 313

**Comment by vishwaikon** [comment]
Hi, @mayura-andrew ,  I fixed the ESLint errors, but the ESLint check still fails.

**Comment by anjula-sack** [comment]
> Hi, @mayura-andrew , I fixed the ESLint errors, but the ESLint check still fails.

@vishwaikon Run npm run lint and see the lint errors should be shown there

**Comment by vishwaikon** [comment]
@mayura-andrew . I ran the command already and fixed the displayed issues for the new files and fixed. Now when run the command shows 10 errors from external files. I'll recheck if I have to fix the configurations.
image

**Pull request closed without merging**